crates/release_plz_core/src/gitea_client.rs

```diff
@@ -13,6 +13,12 @@ pub struct GiteaClient<'a> {
     client: reqwest::Client,
 }
 
+#[derive(Serialize)]
+pub struct CreateReleaseOption<'a> {
+    tag_name: String,
```

Comment:
why this field is a string, while the other is an &str?
Why aren't them of the same type? 🤔